include/constants/field/window.h

```diff
@@ -0,0 +1,7 @@
+#ifndef POKEPLATINUM_CONSTANTS_FIELD_WINDOW_H
+#define POKEPLATINUM_CONSTANTS_FIELD_WINDOW_H
+
+#define FIELD_WINDOW_PALETTE_INDEX  11
+#define FIELD_MESSAGE_PALETTE_INDEX 13
```

Comment:
I'm not 100% on these yet but I wanted to go ahead and push the rest of the changes